core/include/sched.h

```diff
@@ -122,7 +122,8 @@ typedef enum {
  * @{
  */
 #define STATUS_ON_RUNQUEUE      STATUS_RUNNING  /**< to check if on run queue:
-                                                 `st >= STATUS_ON_RUNQUEUE`   */
+                                                     `st >= STATUS_ON_RUNQUEUE`
+                                                 */
```

Comment:
This looks weird. Any idea why uncrustify is doing this?

Reply:
I think actually I did this, must be a merge leftover. Will fix.

Reply:
done. Now uncrustify just realignes the second line a bit.